packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -342,6 +343,10 @@ class Doctor {
             hangingIndent = 0;
             indent = 6;
           }
+          if (message.contextUrl != null) {
+            final String prefix = globals.terminal.supportsEmoji ? '🔨 ' : ' ';
```

Comment:
@zanderso WDUT?

Reply:
The Logger already does the emoji support check, so you can just use it directly here.

Reply:
TIL